src/backend/distributed/connection/worker_log_messages.c

```diff
@@ -57,8 +62,13 @@ static const char *LogLevelNames[] = {
 
 /* postgres log level values corresponding to LogLevelNames */
 static const int LogLevels[] = {
+	DEBUG5,
```

Comment:
this seems sketchy. In this file, the value of DEBUG5 is 0, in all other files it is 10 (as defined in utils/elog.h).

Does LogLevelToLogLevelName give the right result?

Reply:
here we actually don't compare indexes, but stored integer value:

```c
const char *
LogLevelToLogLevelName(int logLevel)
{
....
		if (logLevel == LogLevels[levelIndex])
		{
			return LogLevelNames[levelIndex];
		}
....
}
```